x-pack/plugins/ml/public/application/explorer/swimlane_container.tsx

```diff
@@ -291,10 +291,6 @@ export const SwimlaneContainer: FC<SwimlaneProps> = ({
       },
       heatmap: {
         grid: {
-          cellHeight: {
-            min: CELL_HEIGHT,
-            max: CELL_HEIGHT,
-          },
```

Comment:
Since the overall chart height is computed based on the desired row height, this would have had no effect. Nor does the UI change from this removal looking at the **Anomaly Explorer** in ml.